extensions-core/caffeine-cache/README.md

```diff
@@ -0,0 +1,41 @@
+Druid Caffeine Cache
```

Comment:
Fixed, and in general cleaned up the readme, and moved it to the correct location.